src/confluent_kafka/avro/cached_schema_registry_client.py

```diff
@@ -109,6 +112,11 @@ def __init__(self, url, max_schemas_per_subject=1000, ca_location=None, cert_loc
         self.url = utils.urldefragauth(self.url)
 
         self._session = s
+        key_password = conf.pop('ssl.key.password', None)
+        self.is_key_password_provided = False
```

Comment:
1. use _.
2. Also do an empty string check.

Assignment of `_is_key_password_provided` can be converted to `self._is_key_password_provided = not key_password` expression.

Reply:
sure